tests/fv3_conf/control_run.IN

```diff
@@ -13,60 +13,133 @@ echo "inputdir=$inputdir,NPX=$NPX"
+  if [ $MODEL_INITIALIZATION = false ]; then
     #read_inc and iau tests restart from fh=24
-    cp ../fv3_control${SUFFIX}/RESTART/coupler.res INPUT/.
-    cp ../fv3_control${SUFFIX}/RESTART/fv_core.res.* INPUT/.
-    cp ../fv3_control${SUFFIX}/RESTART/fv_srf_wnd.res.* INPUT/.
-    cp ../fv3_control${SUFFIX}/RESTART/fv_tracer.* INPUT/.
-    cp ../fv3_control${SUFFIX}/RESTART/phy_data.* INPUT/.
-    cp ../fv3_control${SUFFIX}/RESTART/sfc_data.* INPUT/.
-    cp ../fv3_control${SUFFIX}/INPUT/fv3_increment.nc INPUT/.
-    cd INPUT
+    CURRDIR=`pwd`
+    FILEDIR=${CURRDIR}/../control${SUFFIX}/RESTART/
+  else
+    #for warm start from initialiation time
+    FILEDIR=@[INPUTDATA_ROOT]/${inputdir}/INPUT_L127_GDAS
+  fi
+
+  if [[ ${IAU_OFFSET} != 0 ]] || [[ $READ_INCREMENT = '.T.' ]]; then
+
+    if [ $MODEL_INITIALIZATION = false ]; then
+      cp ${FILEDIR}/../INPUT/fv3_increment.nc INPUT/.
+      cp ${FILEDIR}/../INPUT/grid_spec.nc INPUT/.
+      cp ${FILEDIR}/../INPUT/*_grid.tile*.nc INPUT/.
+      cp ${FILEDIR}/../INPUT/oro_data.tile*.nc INPUT/.
+    else
+      cp ${FILEDIR}/fv_increment*.nc INPUT/.
+      cp ${FILEDIR}/grid_spec.nc INPUT/.
+      cp ${FILEDIR}/*_grid.tile*.nc INPUT/.
+      cp ${FILEDIR}/oro_data.tile*.nc INPUT/.
+    fi
+    cp ${FILEDIR}/coupler.res INPUT/.
+    cp ${FILEDIR}/fv_core.res.* INPUT/.
+    cp ${FILEDIR}/fv_srf_wnd.res.* INPUT/.
+    cp ${FILEDIR}/fv_tracer.* INPUT/.
+    cp ${FILEDIR}/phy_data.* INPUT/.
+    cp ${FILEDIR}/sfc_data.* INPUT/.
   else
     #restart test start from fh=12
-    cp ../fv3_control${SUFFIX}/RESTART/20161003.120000.* INPUT/.
+    rsync -arv ${FILEDIR} INPUT/
+    cp ${FILEDIR}/20210322.180000.* INPUT/.
+    cp ${FILEDIR}/../INPUT/grid_spec.nc INPUT/.
+    cp ${FILEDIR}/../INPUT/*_grid.tile*.nc INPUT/.
+    cp ${FILEDIR}/../INPUT/oro_data.tile*.nc INPUT/.
     cd INPUT
-    for RFILE in 20161003.120000.*; do
+    for RFILE in 20210322.180000.*; do
       [ -e $RFILE ] || exit 1
-      mv $RFILE ${RFILE#20161003.120000.}
+      mv $RFILE ${RFILE#20210322.180000.}
     done
+    cd ..
   fi
-  cp ../../fv3_control${SUFFIX}/INPUT/grid_spec.nc .
-  cp ../../fv3_control${SUFFIX}/INPUT/*_grid.tile*.nc .
-  cp ../../fv3_control${SUFFIX}/INPUT/oro_data.tile*.nc .
-  cd ..
 
 fi
 
 cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/aerosol.dat .
 cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/co2historicaldata_201*.txt .
 cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/sfc_emissivity_idx.txt .
 cp    @[INPUTDATA_ROOT]/${inputdir}/INPUT/solarconstant_noaa_an.txt .
-if [ $OZ_PHYS_NEW = .T. ]; then
-  cp  @[INPUTDATA_ROOT]/${inputdir}/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77 ./global_o3prdlos.f77
-elif [ $OZ_PHYS_OLD = .T. ]; then
-  cp  @[INPUTDATA_ROOT]/${inputdir}/INPUT/global_o3prdlos.f77 .
-fi
-if [ $H2O_PHYS = .T. ]; then
-  cp  @[INPUTDATA_ROOT]/FV3_input_data/global_h2o_pltc.f77 ./global_h2oprdlos.f77
-fi
+cp    @[INPUTDATA_ROOT]/${inputdir}/global_o3prdlos.f77 .
```

Comment:
Does this mean that the new (2015) ozone input file, previously called `ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77`, is now the only option/file available with name `global_o3prdlos.f77`? If so, should we remove the "old" 2006 ozone physics from the repository at some point later?

Reply:
@yangfanglin Another option in the future is to use CDEPS to provide the forcing for AMIP-type experiments.